contracts/VeJoeStaking.sol

```diff
@@ -90,7 +88,8 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
         veJoe = _veJoe;
         baseGenerationRate = _baseGenerationRate;
         lastRewardTimestamp = block.timestamp;
-        PRECISION = 1e18;
+        ACC_VEJOE_PER_SHARE_PRECISION = 1e18;
+        GENERATION_RATE_PRECISION = 1e18;
```

Comment:
I separated these two in case we end up wanting to modify them in the future. if we feel confident that we won't, i'm happy to combine them back into just one `PRECISION` variable

Reply:
Works for me